### PR TITLE
sitter profile page

### PR DESCRIPTION
**Pull request by darothub**
### What this PR does (required):
- User can see sitter profile
- User can see gallery of sitter

### Screenshots / Videos (required):
Screenshot 2021-09-06 at 11 15 31 PM

### Any information needed to test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -23,7 +24,7 @@ function App(): JSX.Element {
                 <Route exact path="/settings/profile/">
                   <Profile />
                 </Route>
-                <Route exact path="/login" component={Login} />
+                <Route exact path="/login" component={SitterDetails} />
```
Comment (ajaytomgeorge) [inline client/src/App.tsx:27]: So you are forwarding login to sitterDetails, for test purposes this is okay, but may I know I you included it in the PR?

File: client/src/pages/SitterDetails/SitterDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import ImageListItem from '@material-ui/core/ImageListItem';
+
+import useStyles from './useStyles';
+
+export default function SitterDetails(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
```
Comment (ajaytomgeorge) [inline client/src/pages/SitterDetails/SitterDetails.tsx:17]: @moffatethan mentioned before _baselines should not be included in components and only in the main parent structure._

**Review by ajaytomgeorge — COMMENTED** [review]
@darothub Looks good, @moffatethan can highlight more on front issues if needed.

File: client/src/App.tsx (hunk 1 above)
Comment (darothub) [reply client/src/App.tsx:27]: Actually, it was an oversight. I was supposed to remove it after previewing on my local machine.

File: client/src/pages/SitterDetails/SitterDetails.tsx (hunk 2 above)
Comment (darothub) [reply client/src/pages/SitterDetails/SitterDetails.tsx]: Removed

**Review by moffatethan — APPROVED** [review]
LGTM, nice job on the front-end testd

**Comment by moffatethan** [comment]
Please address the merge conflictd

**Pull request merged**